packages/dduf/src/check-dduf.ts

```diff
@@ -0,0 +1,217 @@
+import { checkFilename } from "./check-filename";
+import { createBlob } from "@huggingface/blob";
+
+export interface DDUFFileEntry {
+	type: "file"; 
+	name: string; size: number; 
+	fileHeaderOffset: number;
+}
+
+export async function* checkDDUF(
+	url: Blob | URL,
+	opts?: { log?: (x: string) => void }
+): AsyncGenerator<{ type: "file"; name: string; size: number; fileHeaderOffset: number }> {
+	const blob = url instanceof Blob ? url : await createBlob(url);
+
+	opts?.log?.("File size: " + blob.size);
+
+	// DDUF is a zip file, uncompressed.
+
+	const last100kB = await blob.slice(blob.size - 100000, blob.size).arrayBuffer();
```

Comment:
```suggestion
	const last100kB = await blob.slice(blob.size - 100_000, blob.size).arrayBuffer();
```
make the digits more readable